### PR TITLE
added the finished homework #8. In order to run the homework and chec…

### PR DESCRIPTION
**Pull request by Valerii-Kukharchuk**
…k it just open homework8/public/index.html in any browser. but it has checked only in Chrome

### CODE REVIEW

File: homework8/public/js/controller.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    function bindUpdateGallery() {
+        gallery.eventHolder.on( gallery.changeEventName, (event, item) => {
+            updateGallery(item);
+        });
+    }
+
+    function updateGallery(searchText) {
+        model.getData(searchText).then((data) => {
```
Comment (ghost) [inline homework8/public/js/controller.js]: data переименовать

File: homework8/public/js/controller.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+    
+    function bindEvents(){
+        //bindSave();  
+        bindUpdate();
+        bindUpdateGallery();
+    }
+    
+    function initGallery(data){
+        gallery = new Gallery(data);
```
Comment (ghost) [inline homework8/public/js/controller.js]: data переименовать

**Review by ghost — CHANGES_REQUESTED** [review]

File: homework8/public/js/controller.js (hunk 2 above)
Comment (Valerii-Kukharchuk) [reply homework8/public/js/controller.js]: fixed

File: homework8/public/js/controller.js (hunk 1 above)
Comment (Valerii-Kukharchuk) [reply homework8/public/js/controller.js]: fixed

**Review by fullstackdb — APPROVED** [review]

**Pull request merged**